unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/Tests/ChatChannelWindowMock.cs

```diff
@@ -0,0 +1,51 @@
+using DCL.Chat.HUD;
+using NSubstitute;
+using System;
+using UnityEngine;
+
+public class ChatChannelWindowMock : MonoBehaviour, IChatChannelWindowView
+{
+    public event Action OnClose;
+    public event Action<bool> OnFocused;
+    public event Action OnBack;
+    public event Action OnRequireMoreMessages;
+    public event Action OnLeaveChannel;
+    public event Action OnShowMembersList;
+    public event Action OnHideMembersList;
+    public event Action<bool> OnMuteChanged;
+    public bool IsActive { get; }
+    public IChatHUDComponentView ChatHUD => Substitute.For<IChatHUDComponentView>();
```

Comment:
This would create a different `IChatHUDComponentView` on every access.

Reply:
This component has been removed, replaced by a Substitute mock